aiven_mysql_migrate/utils.py

```diff
@@ -49,13 +62,26 @@ def from_uri(uri: str, name: Optional[str] = None):
         port = res.port or DEFAULT_MYSQL_PORT
         options = parse_qs(res.query)
         ssl = not (options and options.get("ssl-mode", ["DISABLE"]) == ["DISABLE"])
+
+        LOGGER.debug("from_uri - sslca:[%s], sslcert:[%s], sslkey:[%s]", sslca, sslcert, sslkey)
         return MySQLConnectionInfo(
-            hostname=res.hostname, port=port, username=res.username, password=res.password, ssl=ssl, name=name
-        )
+            hostname=res.hostname,
+            port=port,
+            username=res.username,
+            password=res.password,
+            ssl=ssl,
+            sslca=sslca,
+            sslcert=sslcert,
+            sslkey=sslkey,
```

Comment:
If ssl files are configured I would also check that files are actually accessible for read and throw `WrongMigrationConfigurationException` otherwise, so we can find the problem early on.

Reply:
file checks added